ipatests/test_integration/test_trust.py

```diff
@@ -106,6 +107,65 @@ def check_range_properties(self, realm, expected_type, expected_size):
         expected_text = 'iparangetype: %s\n' % expected_type
         assert expected_text in result.stdout_text
 
+    def add_idrange_auto_private_group(
+        self, rangename, trust_type='ipa-ad-trust', auto_priv_grp_val='false'
+    ):
+        """
+        This function reads an existing idrange added, and creates a new
+        range in order to avoid overlapping values.
+        """
+        add_range = 100000
+        existing_range_name = self.existing_rangename
+        domainname = self.ad_domain.upper()
+        range_dn = DN(
+            ("cn", existing_range_name),
+            ("cn", "ranges"),
+            ("cn", "etc"),
+            self.master.domain.basedn,
+        )
+        conn = self.master.ldap_connect()
+        entry = conn.get_entry(range_dn)  # pylint: disable=no-member
+        baseid = entry.single_value.get("ipaBaseID")
+        rangesize = entry.single_value.get("ipaIDRangeSize")
+        cal_baseid = int(baseid) + int(range) + add_range
+        newbaseid = str(cal_baseid)
+        cmd = [
+            "ipa",
+            "idrange-add",
+            rangename,
+            "--base-id",
+            newbaseid,
+            "--range-size",
+            rangesize,
+            "--rid-base",
+            newbaseid,
+            "--dom-name",
+            domainname,
+        ]
+
+        cmd.extend(("--type", trust_type))
+        cmd.extend(("--auto-private-groups", auto_priv_grp_val))
+
+        self.master.run_command(cmd)
+        self.master.run_command(["systemctl", "restart", "sssd"])
+
+    def mod_idrange_auto_private_group(
+        self, rangename, auto_priv_grp_val='false'
```

Comment:
The method `mod_idrange_auto_private_group` is always called with `rangename = self.existing_rangename`. You can remove this parameter from the method signature and replace `rangename` inside the method with `self.existing_rangename`:
```
    def mod_idrange_auto_private_group(
        self, auto_priv_grp_val='false'):
```